akka-actor/src/main/scala/akka/japi/JavaAPI.scala

```diff
@@ -60,7 +60,7 @@ trait Predicate[T] {
  * Additional tuple types for 3 to 22 values are defined in the `akka.japi.tuple` package, e.g. [[akka.japi.tuple.Tuple3]].
  */
 @SerialVersionUID(1L)
-case class Pair[A, B](first: A, second: B) {
+case class Pair[+A, +B](first: A, second: B) {
```

Comment:
Why was this change introduced? This won't fly because the type is for Java, which does not have declaration-site variance.